### PR TITLE
MUMUP-2045 add description for MyUW logo

### PR DESCRIPTION
**Pull request by jiayinjx**
in addition, avoid some screen reader read out "my uiow"

### CODE REVIEW

File: src/main/webapp/portal/main/partials/header.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
                  <span class="icon-bar"></span>
              </button>
              <div class="header-mobile">
-               <h3><a class="header-text" href="./" id="myuw-header"><img ng-src="{{::headerCtrl.crest}}" class="hidden-xs" alt="{{::headerCtrl.crestalt}}"><span>{{NAMES.title}}</span><span class="beta-logo">{{NAMES.sublogo}}</span></a></h3>
+               <h3><a aria-label="My U W" class="header-text" href="./" id="myuw-header"><img ng-src="{{::headerCtrl.crest}}" class="hidden-xs" alt="{{::headerCtrl.crestalt}}"><span>{{NAMES.title}}</span><span class="beta-logo">{{NAMES.sublogo}}</span></a></h3>
```
Comment (apetro) [inline src/main/webapp/portal/main/partials/header.html]: This looks like the right text to be adding, but it should probably be externalized into `NAMES.title-aria-label` or so, so as to be externalized parallel to `NAMES.title`.

Comment (timlevett) [inline src/main/webapp/portal/main/partials/header.html]: seconded.

**Comment by timlevett** [comment]
What Jira is this related to? Please add it to the title

**Comment by jiayinjx** [comment]
update to NAMES.title-aria-label="My U W" as Andrew suggests.
From the MUM Accessibility Issue log, might not have a Jira key.

**Comment by apetro** [comment]
My bad: hadn't been digested into the JIRA backlog yet, now is as [MUMUP-2045](https://jira.doit.wisc.edu/jira/browse/MUMUP-2045).

**Comment by timlevett** [comment]
Hmmm, I think @apetro 's suggestion was misinterpreted.

Please change it to something like `aria-label="{{NAMES.ariaLabelTitle}}"`

**Comment by jiayinjx** [comment]
add description for MyUW logo - add NAME.ariaLabelTitle in another push request

**Comment by timlevett** [comment]
Related to #9

**Comment by timlevett** [comment]
:+1:

**Pull request merged**